posts/2020/03/Spyder-4.md

```diff
@@ -0,0 +1,31 @@
+<!--
+.. title: Spyder 4 is here!
+.. slug: spyder-4-is-here
+.. date: 2020-03-05 14:00:00 UTC-05:00
+.. author: Carlos Córdoba
+.. tags: Labs, Spyder
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+After more than three years in development, and comprising more than 5000 commits from 60 authors around the world, Spyder 4 finally saw the light in December 5th 2019! I decided to wait until now to make a formal announcement because quickly after the release we found several critical performance issues and some regressions with respect to Spyder 3, most of which are fixed now in version [4.1.0](https://github.com/spyder-ide/spyder/releases), released on <...>.
+
+<!-- TEASER_END -->
+
+This new release comes with a lengthy list of user-requested features aimed at providing users an enhanced development experience at the level of top general-purpose editors and IDEs, while strengthening Spyder's specialized focus on scientific programming in Python. The interested reader can take a look at some of them in [previous](https://labs.quansight.org/blog/2019/11/variable-explorer-improvements-in-Spyder-4/) [blog](https://labs.quansight.org/blog/2019/11/File-management-improvements-in-Spyder4/) [posts](https://labs.quansight.org/blog/2019/08/spyder-40-beta4-kite-integration-is-here/), and in detail in our [Changelog](https://github.com/spyder-ide/spyder/blob/master/CHANGELOG.md#version-400-2019-12-06). However, this post is not meant to describe these improvements but to acknowledge all people that contributed to make Spyder 4 possible.
+
+Spyder 4 is the first version developed with financial support from several companies as well as donations by the international user community. But, as a project, we couldn't have been able to reach the level of maturity needed to receive and handle that support, without the pivotal opportunities Travis Oliphant, former CEO of Anaconda and current one of Quansight, gave me to work in Scientific Python. Thanks to him, I became part of Anaconda, Inc. in 2015, I was able to hire a small small team to improve Spyder within Anaconda in late 2016, and I entered Quansight to work solely on Spyder in 2018. As with other projects in our ecosystem, such as [Bokeh](https://github.com/bokeh/bokeh), [Dask](https://github.com/dask/dask) and [Numba](https://github.com/numba/numba), Spyder benefited immensely from Travis' trust in the role these projects and ours could play in the future. He certainly believed in the vision their maintainers worked so hard to make a reality of, even if their beginnings were humble and their chances of survival uncertain. Therefore, my first big acknowledgment is to Travis: thanks for giving us a chance!
```

Comment:
```suggestion
Spyder 4 is the first version developed with financial support from multiple companies, as well as donations by the international user community. However, as a project, we couldn't have been able to reach the level of maturity needed to receive and handle that support without the pivotal opportunities Travis Oliphant, former CEO of Anaconda and current leader of Quansight, gave me to work in Scientific Python. Thanks to him, I became part of Anaconda Inc. in 2015; I was able to hire a small small team to improve Spyder within Anaconda in late 2016; and I was hired by Quansight to work solely on Spyder in 2018. As with other projects in our ecosystem, such as [Bokeh](https://github.com/bokeh/bokeh), [Dask](https://github.com/dask/dask) and [Numba](https://github.com/numba/numba), Spyder benefited immensely from Travis' trust in the role these efforts and ours could play in the future. He certainly believed in the vision their maintainers worked so hard to make a reality, even if their beginnings were humble and their chances of survival uncertain. Therefore, my first big acknowledgment is to Travis: thanks for giving us a chance!
```

I thought Travis himself was the original developer/maintainer of Numba?

This is _very_ well written.